test/functional/ui/float_spec.lua

```diff
@@ -9285,10 +9292,84 @@ describe('float window', function()
         screen:expect([[
           Ut enim ad minim veniam, quis nostrud             |
           exercitation ullamco laboris nisi ut aliquip ex   |
-          ea co{20:┌}{24:Title}{20:──────────┐}Duis aute irure dolor in    |
-          repre{20:│}{5:popup}{6:it i}{5:text}{20:lu│}tate velit esse cillum      |
-          dolor{20:│}{21:~}{20:eu fugiat null│} pariatur. Excepteur sint   |
-          occae{20:│}{21:~}{20:t cupidatat no│} proident, sunt in culpa    |
+          ea co{26:┌}{27:Title}{26:──────────┐}Duis aute irure dolor in    |
+          repre{26:│}{5:popup}{6:it i}{5:text}{20:lu}{26:│}tate velit esse cillum      |
+          dolor{26:│}{21:~}{20:eu fugiat null}{26:│} pariatur. Excepteur sint   |
+          occae{26:│}{21:~}{20:t cupidatat no}{26:│} proident, sunt in culpa    |
+          {16:^qui o}{22:└}{25:Footer}{22:─────────┘}{16:ollit anim id est           }|
+          laborum.                                          |
+                                                            |
+        ]])
+      end
+
+      -- issue #18576
+      command('hi Nomral guibg=NONE ctermbg=NONE')
+      api.nvim_win_set_option(win, 'winhighlight', 'Normal:Normal')
+      api.nvim_set_option_value('winblend', 100, { win = win })
```

Comment:
can this be added to the `Check that 'winblend' applies to border/title/footer` test case above? The two test cases don't seem to conflict with each other. That would avoid 70 lines of new test code.